task/trigger-jenkins-job/0.1/README.md

```diff
@@ -0,0 +1,103 @@
+# Trigger Jenkins Job
+
+The following task can be used to trigger a Jenkins pipeline using CURL request from a Tekton Task.
+
+More details on Remote Access API can be found [here](https://www.jenkins.io/doc/book/using/remote-access-api/)
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/trigger-jenkins-job/trigger-jenkins-job.yaml
```

Comment:
I think the path needs to be corrected, version directory is missing.

It should be :

`kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/trigger-jenkins-job/0.1/trigger-jenkins-job.yaml`